fundamentals-recap/conditionals-recap.md

```diff
@@ -143,6 +143,15 @@ Conditional expressions can grow in size and complexity. How do we determine tha
 
 Generally, conditional expressions are evaluated from left-to-right, and each expression with one operator resolves first before moving to the next operator.
 
+### !callout-danger
+
+## Errata Notice
```

Comment:
I feel like "errata" is kind of an obscure word that not all students might be familiar with.

Maybe:
```suggestion
## Correction Notice
```